foundation_cms/legacy_apps/wagtailpages/tests/buyersguide/test_homepage.py

```diff
@@ -230,7 +230,7 @@ def test_get_context_no_products(self):
         request = self.request_factory.get(self.bg.url)
         request.user = AnonymousUser()
         request.LANGUAGE_CODE = "en"
-        query_number = 8
```

Comment:
It appears that the updates to the mixin introduced 2 new queries. Needed to update the count here so the tests pass.